www/src/css/lib/vanilla-overrides.scss

```diff
@@ -5,4 +5,50 @@
 // If any styles from vanilla framework need to be overridden, corresponding
 // SCSS rules should be put in this file (not mixed with other styles).
 
+// scss/modules/_header.scss
+.banner .nav-primary ul li:hover {
+  background-color:#f7f7f7;
+}
 
+.banner .nav-primary ul li a,
+.banner .nav-primary ul li a:link,
+.banner .nav-primary ul li a:visited {
+  background-color:transparent;
+  font-size:initial;
+  border-left:0;
+  padding:14px 12px 8px 12px;
+}
+
+.banner .nav-primary ul li a:hover,
+.banner .nav-primary ul li a:link:hover,
+.banner .nav-primary ul li a:visited:hover {
```

Comment:
@stevenwilkin That's a pity. Yes, I see now that this is exactly how vanilla generated selectors look like (which to me looks like a bug). But in such case it has to stay like this.

Maybe it would be worth adding a short one line comment there to explain that these selectors are based on vanilla and are needed for override. So the next person looking there won't have to think about it.